tools/kcli/kcli/main.py

```diff
@@ -1,38 +1,37 @@
 #!/usr/bin/env python
 
-import sys
-from logging import WARNING, INFO, DEBUG
+import logging
 import os
 import re
+import sys
 
 import yaml
-from configure import Configuration
+import configure
 
 from kcli import conf
 from kcli.execute.execute import PLAYBOOKS
-from logger import get_logger
-from parse import create_parser
+from kcli import logger
+from kcli import parse
 
 SETTING_FILE_EXT = ".yml"
 kcli_conf = conf.config
-logger = None
+LOG = logger.LOG
 
 # Representer for Configuration object
 yaml.SafeDumper.add_representer(
-    Configuration,
+    configure.Configuration,
     lambda dumper, value:
     yaml.representer.BaseRepresenter.represent_mapping
     (dumper, u'tag:yaml.org,2002:map', value))
 
 
 def dict_lookup(dic, key, *keys):
-    if logger:
-        calling_method_name = sys._getframe().f_back.f_code.co_name
-        current_method_name = sys._getframe().f_code.co_name
-        if current_method_name != calling_method_name:
-            full_key = list(keys)
-            full_key.insert(0, key)
-            logger.debug("looking up the value of \"%s\"" % ".".join(full_key))
+    calling_method_name = sys._getframe().f_back.f_code.co_name
```

Comment:
The following 6 lines are for debug purpose only.
Please insert those lines into if block and use getEffectiveLevel() to check the logger level as condition